### ISSUE TITLE
Default way to save and load tasks

### ISSUE THREAD

**Issue by xamgore** [natasha/ipyannotate#6]
Hi! Is it possible to save a widget's state to a notebook?

> A Jupyter widget could not be displayed because the widget state could not be found. This could happen if the kernel storing the widget is no longer available, or if the widget state was not saved in the notebook. You may be able to create the widget by running the appropriate cells.

Recently I annotated about 2000 samples, results were't saved :disappointed:

**Comment by kuk** [comment]
Serialise and save `annotation.tasks` to disk manually, for example:

```
labels = {}
for task in annotation.tasks:
    if task.value is not None:
        labels[task.output.host] = task.value

with open('labels.json', 'w') as file:
    json.dump(labels, file)
```

**Comment by xamgore** [comment]
Well, I've tried to load values from json back to `annotation.tasks` (via `Task.set()` method), and preview is not updated. How this can be overcomed?

Imagine: a user is annotating samples, then suddenly the jupyter server shuts down. It would be great, if the user could restart the server and continue annotation process, without extra work (like dumping / loading).

**Comment by kuk** [comment]
Try to load tasks like this:
```
from ipyannotate.buttons import (
    ValueButton as Button,
    NextButton as Next,
    BackButton as Back
)
from ipyannotate.toolbar import Toolbar
from ipyannotate.tasks import Task, Tasks
from ipyannotate.canvas import OutputCanvas
from ipyannotate.annotation import Annotation

labels = load_json(...)
tasks = Tasks(Task(_, value=labels.get(_.key)) for _ in data)
buttons = [
    Button('sexshop', shortcut='1'),
    ...
]
controls = [
    Back(),
    Next()
]
toolbar = Toolbar(buttons + controls)
canvas = OutputCanvas(display=show)
annotation = Annotation(toolbar, tasks, canvas=canvas)
annotation
```

Actually `task.set` method should also work. Try to click `next`, interface should be updated. Universal loading, dumping is hard to implement, annotation data is different in each case (images, html, text).

**Comment by xamgore** [comment]
Even the `pickle` module can't help here?

**Comment by kuk** [comment]
Yep, `pickle` is not universal serializer. But, I think you are right, we need to think about some default way to save and load tasks.

**Comment by xamgore** [comment]
There is a builtin [storemagick extension](https://ipython.readthedocs.io/en/stable/config/extensions/storemagic.html) for IPython, with which users can save & restore data directly from IPython's database (`ipynb` files?). I think it worths mentioning in examples where user works with a simply structured data.
